src/builddate.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+class BuildDate {
```

Comment:
Why is this an extra class? Maybe we should make a `gitinfo.h.in` stead, so that we split off data that changes each commit (git describe, commit date, etc.) from data that is fixed most of the time (CMAKE_PROJECT_VERSION).

That would allow to include the regular `version.h` in `defs_urls.h` so that we can use `CMAKE_PROJECT_VERSION_MAJOR` and `CMAKE_PROJECT_VERSION_MINOR` in the manual URL.